sql_generators/glean_usage/templates/metrics_templating.yaml

```diff
@@ -12,23 +12,23 @@ fenix:
 firefox_ios:
   uri_count:
     # Placeholder for metric that is required for cross-app reporting
-    sql: "SUM(CAST(NULL AS int64))"
+    sql: "SUM(metrics.counter.tabs_normal_and_private_uri_count)"
     counter: true
   is_default_browser:
     sql: "LOGICAL_OR(metrics.counter.app_opened_as_default_browser > 0)"
 focus_ios:
   uri_count:
     # Placeholder for metric that is required for cross-app reporting
```

Comment:
The "Placeholder" comment can now be removed for all the metrics that you defined here